File: app/src/main/java/com/farshidabz/spnote/SPNoteApplication.java
package com.farshidabz.spnote;

import android.app.Application;

import com.google.android.gms.ads.MobileAds;

public class SPNoteApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        MobileAds.initialize(this, getString(R.string.ad_mob_api_key));
    }
}

File: app/src/main/java/com/farshidabz/spnote/SPNoteApplication.kt
package com.farshidabz.spnote;

import android.app.Application;

// import com.google.android.gms.ads.MobileAds; // Commented: AdMob SDK removed to allow build without legacy Google Play services Ads

public class SPNoteApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        // MobileAds.initialize(this, getString(R.string.ad_mob_api_key));
        // Commented: AdMob SDK removed to allow build without legacy Google Play services Ads
    }
}
